docs/pages/localization/Date-fns-customized.example.jsx

```diff
@@ -0,0 +1,106 @@
+import format from 'date-fns/format';
+import frLocale from 'date-fns/locale/fr';
+import ruLocale from 'date-fns/locale/ru';
+import enLocale from 'date-fns/locale/en-US';
+import DateFnsUtils from '@date-io/date-fns';
+import MoreIcon from '@material-ui/icons/MoreVert';
+import React, { useState, useCallback } from 'react';
+import { IconButton, Menu, MenuItem } from '@material-ui/core';
+import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
+
+const localeMap = {
+  en: enLocale,
+  fr: frLocale,
+  ru: ruLocale,
+};
+
+class RuLocalizedUtils extends DateFnsUtils {
+  getCalendarHeaderText(date) {
+    return format(date, 'LLLL', { locale: this.locale });
+  }
+
+  getDatePickerHeaderText(date) {
+    return format(date, 'dd MMMM, EEEE', { locale: this.locale });
+  }
+}
+
+class FrLocalizedUtils extends DateFnsUtils {
+  getDatePickerHeaderText(date) {
+    return format(date, 'd MMM yyyy', { locale: this.locale });
+  }
+}
+
+const localeUtilsMap = {
+  en: DateFnsUtils,
+  fr: FrLocalizedUtils,
+  ru: RuLocalizedUtils,
+};
+
+const localeFormatMap = {
+  en: 'MMMM d, yyyy',
+  fr: 'd MMM yyyy',
+  ru: 'd MMMM yyyy',
```

Comment:
```suggestion
  ru: 'd MMM yyyy',
```

I think it was not expected 
![image](https://user-images.githubusercontent.com/16926049/69334158-ca627580-0c62-11ea-87a1-c81d68d934e4.png)

Reply:
BTW I would expect чет, `21 ноября` or just `21 ноября` in the header title as a Russian speaker. But it is just like an enhancement